apps/ex_nvr_web/lib/ex_nvr_web/live/device_live.html.heex

```diff
@@ -45,6 +45,20 @@
         />
       </div>
 
+      <.input
+        :if={@device_type == "ip"}
+        field={@device_form[:vendor]}
+        id="device_vendor"
+        type={if Mix.env() == :test, do: "text", else: "select"}
```

Comment:
Don't use `Mix`, it's not available on prod releases. No need to use it at all (just use `select` all the time)